source/manuals/logit-io-leavers.html.md.erb

```diff
@@ -0,0 +1,23 @@
+---
+title: How to remove access to Logit
+---
+
+# <%= current_page.data.title %>
+
+When people leave GDS their G Suite account is cancelled, so they'll lose access to Logit but the actual Logit account will not be deleted.
+
+It's important that when people leave GDS, you also delete their logit account so it's clear who in GDS has access.
+
+To remove access to Logit:
+
+**do we need to give instructions on this?**
```

Comment:
I don't think we need instructions for doing this, it's fairly clear in the Logit admin interface.